tests/test_cfn.py

```diff
@@ -39,6 +39,38 @@ def test_query(self):
         resources = p.run()
         self.assertEqual(resources, [])
 
+    def test_disable_protection(self):
+        factory = self.replay_flight_data('test_cfn_disable_protection')
+        p = self.load_policy({
```

Comment:
none of these tests seems like their actually testing anything in the pr, ie. the action implementation could be replaced with a no-op and they would still pass.